src/Translatable/Document/Repository/TranslationRepository.php

```diff
@@ -181,7 +181,7 @@ public function findObjectByTranslatedField($field, $value, $class)
             if ($result instanceof Iterator) {
                 $result = $result->toArray();
             }
-            $id = count($result) ? $result[0]['foreignKey'] : null;
+            $id = count($result) ? $result[0]['foreign_key'] : null;
```

Comment:
```suggestion
            $id = $result[0]['foreign_key'] ?? null;
```

Reply:
Thanks! I've also refactored the body of the method to remove nested `if`s.